Content.Server/Silicons/Laws/SiliconLawSystem.cs

```diff
@@ -152,33 +185,22 @@ protected override void OnGotEmagged(EntityUid uid, EmagSiliconLawComponent comp
 
         base.OnGotEmagged(uid, component, ref args);
         NotifyLawsChanged(uid, component.EmaggedSound);
-        EnsureEmaggedRole(uid, component);
+        EnsureSubvertedSiliconRole(uid);
```

Comment:
this currently throws an error
you need to call it on the mind entity, not on the borg entity